tutorial/transport/py_transport.ipynb

```diff
@@ -383,7 +365,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scen.solve(model='dantzig')"
+    "from tutorial.transport.linopy_model import create_dantzig_model, store_dantzig_solution\n",
+    "\n",
+    "m = create_dantzig_model(i=i, j=j, a=a, b=b, d=d, f=f)\n",
+    "m.solve(\"highs\")\n",
+    "\n",
+    "store_dantzig_solution(model=m, z=z, x=x, demand=demand, supply=supply)"
```

Comment:
Suggest to rename that method, and as before, the Dantzig model code knows which variables and equations to read.
```suggestion
    "read_dantzig_solution(run, model=m)"
```

Reply:
That was how I originally named it and then I thought: the more important part is probably storing it with the `Run`. But I can change back of course ;)

Reply:
Yes, please.